cardano-wallet.cabal

```diff
@@ -32,13 +32,19 @@ library
     ghc-options:         -Werror
   build-depends:
       base
+    , aeson
+    , basement
     , binary
     , bytestring
+    , cardano-crypto
+    , cryptonite
     , cborg
     , containers
     , cryptonite
+    , data-default
```

Comment:
Let's not, we don't need this. A plain `defaultWhatever` function is fine.

Reply:
removed data-default